tests/testing_support/external_fixtures.py

```diff
@@ -136,6 +136,23 @@ def validate_cross_process_headers(wrapped, instance, args, kwargs):
 
     return result
 
+
+@function_wrapper
+def validate_messagebroker_headers(wrapped, instance, args, kwargs):
+    result = wrapped(*args, **kwargs)
+
+    transaction = current_transaction()
+    settings = transaction.settings
+
+    if settings.distributed_tracing.enabled:
+        validate_distributed_tracing_header()
+    else:
+        validate_outbound_headers(header_id='NewRelicID',
+                header_transaction='NewRelicTransaction')
+
+    return result
+
+
```

Comment:
Shouldn't this be moved into its own file in the validators folder? We had discussed this in mobbing at some point recently but I don't remember if you were here for that or not.